ps_buybuttonlite.php

```diff
@@ -0,0 +1,197 @@
+*  International Registered Trademark & Property of PrestaShop SA
+*/
+
+if (!defined('_PS_VERSION_')) {
+    exit;
+}
+
+class Ps_BuybuttonLite extends Module
+{
+    public function __construct()
+    {
+        $this->name = 'ps_buybuttonlite';
+        $this->tab = 'administration';
+        $this->version = '1.0.0';
+        $this->author = 'PrestaShop';
+        $this->need_instance = 0;
+
+        $this->module_key = 'bab535c26e031e9d224c0d128e063283';
+
+        $this->bootstrap = true;
+
+        parent::__construct();
+
+        $this->controllerAdmin = 'AdminAjaxPs_buybuttonlite';
+
+        $this->displayName = $this->trans('Buy button lite', array(), 'Modules.Buybuttonlite.Admin');
+        $this->description = $this->trans('Create a product checkout redirect link to post on a blog or any social media', array(), 'Modules.Buybuttonlite.Admin');
+        $this->ps_version = (bool)version_compare(_PS_VERSION_, '1.7', '>=');
+
+        // Settings paths
+        $this->css_path = $this->_path.'views/css/';
+
+        // Confirm uninstall
+        $this->confirmUninstall = $this->trans('Are you sure you want to uninstall this module?', array(), 'Modules.Buybuttonlite.Admin');
+        $this->ps_versions_compliancy = array('min' => '1.7', 'max' => _PS_VERSION_);
+    }
+
+    /**
+     * install()
+     *
+     * @param none
+     * @return bool
+     */
+    public function install()
+    {
+        if (parent::install() &&
+            $this->installTab()) {
+                return true;
+        }
+        return false;
+    }
+
+    /**
+     * uninstall()
+     *
+     * @param none
+     * @return bool
+     */
+    public function uninstall()
+    {
+        if (parent::uninstall() &&
+            $this->uninstallTab()) {
+            return true;
+        }
+        return false;
+    }
+
+    /**
+     * Register admin controller (ajax call)
+     *
+     * @param none
+     * @return bool
+     */
+    public function installTab()
+    {
+        $tab = new Tab();
+        $tab->class_name = $this->controllerAdmin;
+        $tab->active = 1;
+        $tab->name = array();
+        foreach (Language::getLanguages(true) as $lang) {
+            $tab->name[$lang['id_lang']] = $this->name;
+        }
+        $tab->id_parent = -1;
+        $tab->module = $this->name;
+
+        return $tab->add();
+    }
+
+    /**
+     * Unregister admin controller
+     *
+     * @param none
+     * @return bool
+     */
+    public function uninstallTab()
+    {
+        $id_tab = (int)Tab::getIdFromClassName($this->controllerAdmin);
+        $tab = new Tab($id_tab);
+        if (Validate::isLoadedObject($tab)) {
+            return ($tab->delete());
+        } else {
+            return false;
+        }
+    }
+
+    /**
+     * Load dependencies
+     *
+     * @param none
+     * @return bool
+     */
+    public function loadAsset()
+    {
+        $css = array(
+            'https://unpkg.com/element-ui/lib/theme-chalk/index.css',
```

Comment:
This might not be the right PR to discuss it but is there a rule about using CDNs ? By doing this you take the risk that if this resource becomes unavailable or is changed your module will be down

A CDN is supposed not to do that of course but I'm still wondering 🐙